tests/unit/src/main/scala/tests/BaseAmmoniteSuite.scala

```diff
@@ -355,7 +355,7 @@ abstract class BaseAmmoniteSuite(scalaVersion: String)
 
   test("simple errored script") {
     val expectedDiagnostics =
-      """main.sc:15:25: error: not found: type Fooz
+      """errored.sc:15:25: error: not found: type Fooz
```

Comment:
It seems using `main` causes an additional warning because there is already `main` improted by ammonite. Not sure if it's really a bug though.